styles/nav/_mixins.scss

```diff
@@ -0,0 +1,41 @@
+@mixin link-tab() {
```

Comment:
No. Main nav tab is different. Or at lease different enough from subnav tab. It is tab that looks like a link. That is has not background on it, really not a box or a tab. I made it a mixin so I can reuse it within ppm, because we need it for user links. I was debating wether to create a seperate class like link-tab and that would require a ton of changes. Do you have other ideas?